### PR TITLE
Add RSS feed with basic spec updates

### PR DESCRIPTION
**Pull request by DanielOaks**
For some time, people have wanted a way to just subscribe for the occasional update on what we're doing. This PR introduces an RSS feed to let people subscribe for the occasional update on what we're doing, the status of our specs, and recent changes.

This also fills out the WG page a fair bit, which is good. Solves #83 and #103.

Example post:
screen shot 2016-11-20 at 12 35 41 pm

And how it's displayed on the WG page:
screen shot 2016-11-20 at 12 35 27 pm

### CODE REVIEW

**Comment by jwheare** [comment]
Great! Let's get a link to the roadmap milestone on the WG page while we're at it. That and status updates/changelog should probably be on the home page too.

**Comment by DanielOaks** [comment]
Yeah threw roadmap link on the WG as 'the top article', should go well. I'm iffy on throwing it on the home page too, just due to how much we already have on there and having to modify this a decent amount to get it to play nicely there. Mind if we leave that 'til after this is merged/in, since it may take a bit of time to play with and see exactly how we wanna tackle that?

File: wg.html [hunk 1]
```diff
@@ -33,3 +33,32 @@
...
+      <p style="margin-bottom: 2rem">Below is a rough log of what the IRCv3 working group's been up to including recent changes to specifications and new proposals:</p>
+
+</div></div>
+
+<div class="container">
+
+      <div class="post">
+          <h3><a href="https://github.com/ircv3/ircv3-specifications/milestone/4">Roadmap</a></h3>
+          <div class="post-content">
+              <p>Our roadmap <a href="https://github.com/ircv3/ircv3-specifications/milestone/4"><sup>[link]</sup></a> always contains info on our direction moving forward, the sort of features we have planned and our priorities.</p>
```
Comment (dequis) [inline wg.html]: `roadmap <a><sup>[link]</sup></a>` -> `<a>roadmap</a>`?

**Comment by jwheare** [comment]
Yeah fine by me to delay home page stuff.

**Comment by DanielOaks** [comment]
Cool, cleaned up the language there and added notes for STS and SNI.

screen shot 2016-11-20 at 2 00 37 pm

Should be good to go with this?

**Comment by ChickenNuggers** [comment]
Is there a reason links are not implemented using just anchor tags for `[link]`s?

**Comment by DanielOaks** [comment]
Because they're a bit difficult to see and this brings them out a bit more. I'll make links in general easier to see but I'd probably prefer having the external links like this (particularly as this is kinda like a reference page, having the explicit external links could make sense here).

**Comment by ChickenNuggers** [comment]
> having the explicit external links could make sense here

Now that you mention it, that's a good point.

**Comment by jwheare** [comment]
LGTM

**Pull request merged**